src/mlpack/methods/preprocess/preprocess_imputer_main.cpp

```diff
@@ -50,8 +50,9 @@ BINDING_SEE_ALSO("@preprocess_binarize", "#preprocess_binarize");
 BINDING_SEE_ALSO("@preprocess_describe", "#preprocess_describe");
 BINDING_SEE_ALSO("@preprocess_split", "#preprocess_split");
 
-PARAM_STRING_IN_REQ("input_file", "File containing data.", "i");
-PARAM_STRING_OUT("output_file", "File to save output into.", "o");
+// Define parameters for data.
+PARAM_MATRIX_IN_REQ("input", "Matrix containing data.", "i");
```

Comment:
Since you renamed the input, you have to update some of the existing tests as well, e.g. https://github.com/mlpack/mlpack/blob/master/src/mlpack/tests/main_tests/preprocess_imputer_test.cpp#L45 to use `input` instead of `input_file`.